src/components/Explore/Explore.tsx

```diff
@@ -98,10 +99,10 @@ const Explore: React.FunctionComponent<{
     <Styles.Container>
       <Grid container spacing={1}>
         <Grid item sm={6} xs={12}>
-          <Styles.Heading variant="h4">Trends</Styles.Heading>
+          <Styles.Heading variant="h4">Raw data</Styles.Heading>
```

Comment:
Huh.  FWIW- I don't love this name. Seems very generic and I don't know if "raw" conveys anything useful.  Maybe "Data Explorer" or something?  I guess there was a discussion about this on slack already though, so if this is the consensus, okay.

Also, we call it the "trends" chart in the update text in the header at the top of the page.  But maybe that's fine. I don't know how long we'll keep that text anyway. 🤷

Reply:
ok, will revert to Trends then so it's consistent with the copy in the update.